torch_xla/core/xla_model.py

```diff
@@ -120,6 +165,11 @@ def xla_replication_devices(local_devices):
 
 
 def set_replication(device, devices):
+  """
+    Sets the current thread's device to device and sets replication devices
```

Comment:
Ditto as for replciation_devices() above.

Reply:
Sounds good. I updated our spreadsheet to reflect that these are undocumented. Should we change private functions' names to start with an underscore?